docs/sources/operators-guide/configure/configure-otel-collector.md

```diff
@@ -0,0 +1,106 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/configuring/configuring-otel-collector/
+description: Learn how to write data from OpenTelemetry Collector into Mimir
+menuTitle: Configuring OTel Collector
+title: Configuring OpenTelemetry Collector to write data into Mimir
+weight: 150
+---
+
+# Configuring OpenTelemetry Collector to write data into Mimir
+
+When using the OpenTelemetry Collector, you can write metrics data into Mimir via two options: `remote_write` and `otlphttp`.
```

Comment:
`remote_write` doesn't match a config option, so I see two options:
1. Rename `remote_write` to `prometheusremotewrite`.
2. Change `remote_write` to "remote write" (without double quotes)

_Same comment applies below._